### PR TITLE
RET-3794: added new field for TSE Respondent response mid event input…

### PR DESCRIPTION
**Pull request by robin-ieong**
New fields for TSE Respondent respond mid event validation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java [hunk 1]
```diff
@@ -1135,6 +1135,9 @@ public class CaseData extends Et1CaseData {
     private String tseResponseCopyNoGiveDetails;
     @JsonProperty("resTseTableMarkUp")
     private String resTseTableMarkUp;
+    //  if Respondent is responding to Tribunal
+    @JsonProperty("tseRespondToTribunal")
```
Comment (Itay-Kirsh) [inline src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java:1139]: Maybe change it to tseRespondingToTribunal

Comment (robin-ieong) [reply src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java:1139]: Any reason why?

Comment (Itay-Kirsh) [reply src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java:1139]: I think it's a better name, it's even what you put in the comment. It's not a deal breaker for me but I think it'd be nicer

**Review by fergusoshea — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java (hunk 1 above)
Comment (robin-ieong) [reply src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java]: updated

**Review by fergusoshea — APPROVED** [review]

**Review by Itay-Kirsh — APPROVED** [review]

**Pull request merged**